### DISCUSSION TITLE
Quickstart cannot be run locally

### DISCUSSION THREAD

**Discussion by tqq1994516** [wasmCloud/wasmCloud#3763] [Q&A] [answered]
When I followed the [Quickstart](https://wasmcloud.com/docs/tour/hello-world)  to perform operations in WSL2 Ubuntu 24.04 until the step of "wash dev", it didn't execute as expected.
![image](https://github.com/user-attachments/assets/58776b49-3306-49f8-b00d-b7bca33a8ade)
![image](https://github.com/user-attachments/assets/949366c9-f8d0-4014-8781-5a58671182c7)

**Comment by brooksmtownsend** [comment]
Hey @tqq1994516 , sorry to hear you're hitting a bug. That error looks like the host is having a hard time starting, would you mind running the following commands to try again?

```bash
wash down --all
wash dev
```

If the error persists, See if you can post the contents of the logs from `~/.wash/dev/<session_id>/*.log` here, your session ID is printed at the top of the log output.

**Reply by tqq1994516** [reply, marked as answer]
I've done some troubleshooting. It should be caused by the incomplete download of the `wasmcloud_host` file. I manually downloaded and replaced it, and now it's working properly.
